CHANGELOG

```diff
@@ -1,6 +1,11 @@
 Unreleased
 ==========
 
+0.3.0 (2019-07-26)
+==================
```

Comment:
The release date gets added once the package is ready to release. For now, this can go under "Unreleased".